tests/ImageSharp.Benchmarks/Color/Bulk/PackFromVector4.cs

```diff
@@ -40,7 +44,7 @@ public void Cleanup()
             this.source.Dispose();
         }
 
-        [Benchmark(Baseline = true)]
+        //[Benchmark]
```

Comment:
Should this still be commented?

Reply:
The benchmark code serves as an information, but the execution is unnecessary, unless someone wants to evaluate that specific method in future investigations.

I wish there was some better way to Skip benchmarks without completely dropping their code.